erpnext/stock/doctype/item/test_item.py

```
@@ -88,11 +106,25 @@ def get_item(self, idx):
 			item = frappe.get_doc("Item", item_code)
 		return item
 
-	def test_get_item_details(self):
-		# delete modified item price record and make as per self.globalTestRecords["Item"]
-		frappe.db.sql("""delete from `tabItem Price`""")
-		frappe.db.sql("""delete from `tabBin`""")
+	def make_bin(self, records):
+		for x in records:
+			x = frappe._dict(x)
+			bin = qb.DocType("Bin")
```

Ruff (0.15.9)

[error] Variable `bin` is shadowing a Python builtin

(A001)